src/NzbDrone.Core/MediaFiles/MediaFileExtensions.cs

```diff
@@ -24,7 +24,8 @@ static MediaFileExtensions()
                 { ".wav", Quality.WAV },
                 { ".wv", Quality.WAVPACK },
                 { ".flac", Quality.FLAC },
-                { ".ape", Quality.APE }
+                { ".ape", Quality.APE },
+                { ".cue", Quality.Unknown }
```

Comment:
The cue file isn't a media file, is there a better way set this up so as to not confuse other parts of the code?

Reply:
Honestly I don't know, I abused this list a bit as the "accepted file extensions" so other file-related services could be modified less to support the cue file.